examples/fastapi/charm/lib/charms/traefik_k8s/v2/ingress.py

```diff
@@ -73,7 +72,7 @@ def _on_ingress_revoked(self, event: IngressPerAppRevokedEvent):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 14
+LIBPATCH = 13
```

Comment:
it looks like we're going backwards on the version here? Is version 14 available, or perhaps an even later version?

Reply:
Since the library is in an example folder, they're not automatically updated, all the other examples are using version 13, so Ali aligned.
I think it should be good to:
- Upgrade the libraries manually once to get the latest
- Add a story to find a way to automate this library upgrade via juju action or renovate

Reply:
Could done in a separate PR as it's unrelated to this PR